integration/tools/validation/src/main/java/alluxio/fuse/correctness/ReadMultiFileValidation.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.correctness;
+
+import static alluxio.fuse.correctness.Utils.RANDOM;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class validates multiple-file read correctness of AlluxioFuse.
+ */
+public class ReadMultiFileValidation {
+  /**
+   * The whole process of validating reading correctness on multiple files.
+   *
+   * @param options the options for the validation test
+   */
+  public static void run(Options options) {
+    for (long fileSize : Constants.FILE_SIZES) {
+      System.out.println(String.format(
+          Constants.TESTING_FILE_SIZE_FORMAT, Constants.READ, fileSize));
+      List<String> localFileList = new ArrayList<>(options.getNumFiles());
+      List<String> fuseFileList = new ArrayList<>(options.getNumFiles());
+      createTestFiles(fileSize, options, localFileList, fuseFileList);
+      for (int bufferSize : Constants.BUFFER_SIZES) {
+        validate(Constants.SEQUENTIAL_READ,
```

Comment:
Can those be put as IOOperation type?

Reply:
Then if user in the command line specify `sequential read`, we should only test sequential read but not random read or mixed read?

Reply:
The current structure is a little complicated
separate those read types may help improve the code structure